cmd/kube-apiserver/app/server.go

```diff
@@ -694,6 +694,19 @@ func defaultOptions(s *options.ServerRunOptions) error {
 		}
 	}
 
+	// TODO: remove when we stop supporting the legacy group version.
+	if s.APIEnablement.RuntimeConfig != nil {
+		for key, value := range s.APIEnablement.RuntimeConfig {
+			if strings.HasPrefix(key, "v1") || strings.HasPrefix(key, "api/v1") {
```

Comment:
to be precise, this must be `key == "xyz" || strings.HasPrefix(key, "abc/")` for each of the two.

Reply:
make sense